Edge 10.18.0 release notes (#380)
SoftwareAG/c8y-releasenotes: content/release-10-18-0/edge-10-18-0-bundle/10_18_0.md

---
weight: 60
title: Release 10.18.0
layout: redirect
---

> The Cumulocity IoT Edge release 10.18 includes new features, improvements, fixes and known issues for Cumulocity IoT 10.18. For more information, see the [What´s New](/release-10-18-0/whatsnew-10-18-0/) section and our video [What's new in Cumulocity IoT 10.18](https://www.youtube.com/watch?v=pTouaQbc6SI&list=PLexm-BA6Wrr8AYnu97vpIR7uicD2wsyms).

Reviewer: ```suggestion
The release notes for **Cumulocity IoT Edge release 10.18.0** contain important information about all the changes, including new features, enhancements, and fixes that have been made to both the **Cumulocity IoT Edge Appliance VM** and the **Cumulocity IoT Edge on Kubernetes**. If you want to know more, you can check out the [What´s New](/release-10-18-0/whatsnew-10-18-0/) section and watch our video [What's new in Cumulocity IoT 10.18](https://www.youtube.com/watch?v=pTouaQbc6SI&list=PLexm-BA6Wrr8AYnu97vpIR7uicD2wsyms).

Additionally, the release notes contain some [announcements](/release-10-18-0/edge-10-18-0/#announcements) regarding planned and implemented deprecations that might require some action on your part at some point.
```

> Download the Cumulocity IoT Edge VM-based distribution version 10.18 from the [Software AG Empower portal](https://empower.softwareag.com), based on the target hypervisor.

Reviewer: ```suggestion

You can find the corresponding documentation for **Cumulocity IoT Edge Appliance VM** at https://cumulocity.com/guides/10.18.0/edge/edge-introduction/ and for **Cumulocity IoT Edge on Kubernetes** at https://cumulocity.com/guides/10.18.0/edge-k8s/k8-edge-introduction/
```

> The release notes for **Cumulocity IoT Edge release 10.18.0** contain important information about all the changes, including new features, enhancements, and fixes that have been made to both the **Cumulocity IoT Edge Appliance VM** and the **Cumulocity IoT Edge on Kubernetes**. If you want to know more, you can check out the [What´s New](/release-10-18-0/whatsnew-10-18-0/) section and watch our video [What's new in Cumulocity IoT 10.18](https://www.youtube.com/watch?v=pTouaQbc6SI&list=PLexm-BA6Wrr8AYnu97vpIR7uicD2wsyms).

Reviewer 2: ```suggestion
The release notes for **Cumulocity IoT Edge release 10.18.0** contain important information about all the changes, including new features, enhancements, and fixes, that have been made to both **Cumulocity IoT Edge Appliance VM** and **Cumulocity IoT Edge on Kubernetes**. If you want to know more, you can check out the [What´s New](/release-10-18-0/whatsnew-10-18-0/) section and watch our video [What's new in Cumulocity IoT 10.18](https://www.youtube.com/watch?v=pTouaQbc6SI&list=PLexm-BA6Wrr8AYnu97vpIR7uicD2wsyms).
```

> Additionally, the release notes contain some [announcements](/release-10-18-0/edge-10-18-0/#announcements) regarding planned and implemented deprecations that might require some action on your part at some point.

Reviewer 2: ```suggestion
Additionally, the release notes contain [announcements](/release-10-18-0/edge-10-18-0/#announcements) regarding planned and implemented deprecations that might require some action on your part at some point.
```

> You can find the corresponding documentation for **Cumulocity IoT Edge Appliance VM** at https://cumulocity.com/guides/10.18.0/edge/edge-introduction/ and for **Cumulocity IoT Edge on Kubernetes** at https://cumulocity.com/guides/10.18.0/edge-k8s/k8-edge-introduction/

Reviewer 2: ```suggestion
You can find the corresponding documentation for **Cumulocity IoT Edge Appliance VM** at https://cumulocity.com/guides/10.18.0/edge/edge-introduction/ and for **Cumulocity IoT Edge on Kubernetes** at https://cumulocity.com/guides/10.18.0/edge-k8s/k8-edge-introduction/.
```

### Cumulocity IoT Edge Appliance VM

#### Rocky Linux 8

> With CentOS 7 reaching end-of-life on June 30th, 2024, we are introducing support for an alternate OS for Edge Appliance users. In this release, Edge Appliance VM comes with [Rocky Linux 8](https://rockylinux.org/). Since the Operating System is changed, the upgrade from earlier version of Edge entails a side-by-side installation accompanied by data migration. For information on upgrading Edge version 10.18, see [Migrating Edge 10.17 to 10.18](https://cumulocity.com/guides/10.18.0/edge/update-bundle/migration-1017-to-1018/).

Reviewer 3: ```suggested
As CentOS 7 will reach its end-of-life on June 30th, 2024, we are transitioning our support to an alternative Operating System for Edge Appliance users. Starting with this release, the Edge Appliance VM will be equipped with [Rocky Linux 8](https://rockylinux.org/). Due to the change in VM operating system, upgrading from previous version of Edge will require a side-by-side installation accompanied by data migration. For details on how to upgrade from Edge version 10.17 to 10.18, please refer to [Migrating Edge 10.17 to 10.18](https://cumulocity.com/guides/10.18.0/edge/update-bundle/migration-1017-to-1018/).
```

### Cumulocity IoT Edge on Kubernetes

> 

Reviewer 3: @bbyreddy shall we mention that 10.18 is GA version of Edge on Kubernetes?

> As CentOS 7 will reach its end-of-life on June 30th, 2024, we are transitioning our support to an alternative Operating System for Edge Appliance users. Starting with this release, the Edge Appliance VM is equipped with [Rocky Linux 8](https://rockylinux.org/). Since the Operating System is changed, the upgrade from earlier version of Edge entails a side-by-side installation accompanied by data migration. For information on upgrading Edge version 10.18, see [Migrating Edge 10.17 to 10.18](https://cumulocity.com/guides/10.18.0/edge/update-bundle/migration-1017-to-1018/).

Reviewer 2: ```suggestion
As CentOS 7 will reach its end-of-life on June 30th, 2024, we are transitioning our support to an alternative Operating System for Edge Appliance users. Starting with this release, the Edge Appliance VM is equipped with [Rocky Linux 8](https://rockylinux.org/). Since the Operating System is changed, the upgrade from earlier versions of Edge entails a side-by-side installation accompanied by data migration. For information on upgrading Edge version 10.18, see [Migrating Edge 10.17 to 10.18](https://cumulocity.com/guides/10.18.0/edge/update-bundle/migration-1017-to-1018/).
```

> Cumulocity IoT Edge on Kubernetes is GA and is now ready for deployment in production environments.

Reviewer 2: ```suggestion
Cumulocity IoT Edge on Kubernetes has reached General Availability (GA) and is now ready for deployment in production environments.
```

#### Remote software management

Using the Cumulocity IoT software management feature, users can now remotely manage applications, microservices, and extensions installed on Edge.

> For detailed instructions, please refer to [Managing software remotely](https://cumulocity.com/guides/10.18.0/edge-k8s/k8-edge-connecting-edge-to-cloud-bundle/software-management-docs).

Reviewer 2: ```suggestion
For detailed instructions, refer to [Managing software remotely](https://cumulocity.com/guides/10.18.0/edge-k8s/k8-edge-connecting-edge-to-cloud-bundle/software-management-docs).
```

### Notice of Deprecation: Usage of Cumulocity Linux Agent in Cumulocity IoT Edge

With the Cumulocity IoT Edge, version 10.18, we are deprecating the out-of-the-box shipment of Cumulocity Linux Agent within the Cumulocity IoT Edge offering, effective 31st of March 2024. The 10.18 will be the last version to include Cumulocity Linux Agent.

We are committed to guiding and supporting you through this transition. For any questions or further assistance, please get in touch with our support team or our Product Manager, Nick Ponomar, at nick.ponomar@softwareag.com.

We appreciate your understanding and cooperation as we continue to enhance our product offerings.

> ### Cumulocity IoT Edge on Kubernetes - Public Preview

Reviewer: ```suggestion
### Cumulocity IoT Edge on Kubernetes
```

> For detailed instructions, please refer to [Monitoring](https://cumulocity.com/guides/10.18.0/edge-k8s/edge-operations-bundle/monitoring).

Reviewer 2: ```suggestion
For detailed instructions, refer to [Monitoring](https://cumulocity.com/guides/10.18.0/edge-k8s/edge-operations-bundle/monitoring).
```
